.github/workflows/run-e2e-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Run end-to-end tests
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches: [master]
+    paths: # add other modules when there are under e2e tests
+      - 'powertools-e2e-tests/**'
+      - 'powertools-core/**'
+      - 'powertools-serialization/**'
+      - 'powertools-logging/**'
+      - 'powertools-tracing/**'
+      - 'powertools-idempotency/**'
+      - 'powertools-metrics/**'
+      - 'pom.xml'
+      - '.github/workflows/**'
+
+jobs:
+  e2e:
```

Comment:
Is there a way to ensure that `cdk destroy` always runs after regardless of what happens with the main process? It is easy to imagine that we quickly end up in a situation where something like github action worker timeouts periodically leave us in a state where we are cleaning up the AWS account by hand periodically.

Reply:
Discussed with Ruben, they don't have any other belt around GitHub potential issue, just around their own code, just like this AfterAll...